mozilla_bitbar_devicepool/main.py

```diff
@@ -71,6 +71,7 @@ def run_test(args):
     configuration.configure(bitbar_configpath, filespath=args.files, update_bitbar=args.update_bitbar)
 
     run_test_for_project(args.project_name)
+    logger.info("run started for project '%s'" % args.project_name)
```

Comment:
This won't be too verbose?

Reply:
This isn't in test-run-manager, but main.py (what we use for starting docker image builds from the CLI).

Reply:
doh! Thanks